### ISSUE TITLE
Include note about built-in datasets in R 

### ISSUE THREAD

**Issue by maneesha** [swcarpentry/r-novice-gapminder#848]
The [list subsetting exercise](https://swcarpentry.github.io/r-novice-gapminder/06-data-subsetting.html#list-subsetting) references the built-in dataset `mtcars`. Consider including  an explanation of what that is and why we did not need to define it as we did all our other datasets.

**Comment by Yatish0833** [comment]
I agree. Built-in R datasets should be introduced. The ability for students to see all datasets available to then using "data()" function is important. Maybe a good place to introduce would be at the very start of [data structures ](https://swcarpentry.github.io/r-novice-gapminder/04-data-structures-part1.html) lesson before building the feline-data.csv.

**Comment by naupaka** [comment]
Good suggestion!
